src/status_im/ui/screens/accounts/login/events.cljs

```diff
@@ -96,22 +96,26 @@
                         wrap-with-stop-node-fx)]
       (wrap-fn db' address password))))
 
+(defn is-new-account?
+  "checks if the account is in the accounts in the db"
+  [{:accounts/keys [accounts]} address]
+  (not ((into #{} (keys accounts)) address)))
```

Comment:
I'd call this `(set (keys accounts))` and put it as an inline let predicate fn, `present?` or something so it is `(not (present? address)`